tests/test_cli.py

```diff
@@ -52,6 +53,13 @@ def test_init(app):
     ])
     assert 6 == search.number_of_indexes
 
+    with patch('invenio_search.ext.iter_entry_points',
+               return_value=template_entrypoints('invenio_search.templates')):
+        assert 'file_download_v1' in search.templates
+        assert 'record_view_v1' in search.templates
+        assert 'subdirectory-file_download_v1' in search.templates
+        assert len(search.templates.keys()) == 3
+
```

Comment:
Could you test here that the templates are correctly registered in elasticsearch.